### ISSUE TITLE
It crashes on snowmen dlc for some reason

### ISSUE THREAD

**Issue by Abrek01** [juso40/bl2sdk_Mods#26]
The title explains all I was trying to play BL2 with the mod and for some reason whenever I kill a snowmen the game gives me a fatal error.
I'll list the mods I'm using since there might be a conflict that causes this:

![mods](https://user-images.githubusercontent.com/128305852/226183812-86dcce6c-94f3-444a-989c-cad2be2252e5.png)
These + UCP 5.0.4

**Comment by boostedmonkey** [comment]
I ran into this issue aswell today.

It seems to be fixed when u just turn off glory kills, no need to disable the entire mod.
It also occurs even when you one shot the mob, so it doesn't even trigger a glory kill on the snowman.

**Comment by ilrathCXV** [comment]
To fix this, you will need to go into the "glorykills.py" and add these "snowminion" lines inside "def enter"glory_kill_state":


```
...
 if "bugmorph" in caller.PathName(caller.MyWillowPawn.AIClass).lower():
            return True
            
 if "snowminion" in caller.PathName(caller.MyWillowPawn.AIClass).lower():
            return True

```

This so far seems to work for the crashing.

**Comment by juso40** [comment]
Yup , @ilrathCXV answer is the correct fix. I added the snowminions to the mods blacklist.

This bug is now fixed in BL2Eternal v1.6
